Resculpt/Resculptable.cs

```diff
@@ -90,6 +90,22 @@ public sealed class Resculptable : KMonoBehaviour {
 			}
 		}
 
+
+		/// <summary>
+		/// Triggered when the user requests a rotation of the decor item.
+		/// </summary>
+		private void OnRotateClicked()
+		{
+			Rotatable rotatable = this.gameObject.GetComponent<Rotatable>();
+			if (rotatable != null) rotatable.Rotate();
+
+			// Buildings with even width values jump one tile when rotating and must be moved back
+			BuildingDef def = this.gameObject.GetComponent<Building>()?.Def;
```

Comment:
`?.` does not work on destroyed components, as it uses `Object.Equals` which Unity breaks. Either explicitly store and check the `Building`, or use (extension) `GetComponentSafe<T>(this GameObject)`.

Reply:
Building is now stored for the check